wiki/en/en-Getting-Started.md

```diff
@@ -39,7 +39,9 @@ Also - [Jamulus OS!](https://sourceforge.net/projects/jamulus-os/files/JamulusOS
 
 ## Recommended hardware
 
-See [the list of known good hardware](Sound-Devices) and [Hardware Setup](Hardware-Setup).
+In general, your device must support a sample rate of **48kHz** (DVD quality). On Windows it's recommended to choose a device with a native [ASIO](https://en.wikipedia.org/wiki/Audio_Stream_Input/Output) driver.
+
```

Comment:
I would add something like this here:
Your device should also allow you to listen to the computer's audio signal. (Jamulus server mix monitoring)

Reply:
Added. Can still be discussed